libfreerdp/core/freerdp.c

```diff
@@ -526,10 +545,17 @@ BOOL freerdp_context_new(freerdp* instance)
 
 	update_register_client_callbacks(rdp->update);
 
+	instance->context->abortEvent = CreateEvent(NULL, TRUE, FALSE, NULL);
+	if (!instance->context->abortEvent)
+		goto out_error_abort_event;
+
 	IFCALLRET(instance->ContextNew, ret, instance, instance->context);
 
 	if (ret)
 		return TRUE;
+
+out_error_abort_event:
```

Comment:
If I'm not wrong the action here is not correct (if we haven't manage to create the event, no need to close it)